cinema/views.py

```diff
@@ -0,0 +1,41 @@
+from django.core.serializers import serialize
+from django.http import JsonResponse
```

Comment:
The import `from django.http import JsonResponse` is not used in the code and can be removed to clean up the imports.